### PR TITLE
feat:FPLA-2302 moved all config files to namespaces directory

### PR DESCRIPTION
**Pull request by nkoranne**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: k8s/namespaces/family-public-law/fpl-case-service/aat.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        MANAGEMENT_ENDPOINTS_WEB_EXPOSURE_INCLUDE: health, info, threaddump
+        GATEWAY_URL: https://gateway-ccd.aat.platform.hmcts.net
+        AUTH_IDAM_CLIENT_BASEURL: https://idam-api.aat.platform.hmcts.net
+        SPRING_SECURITY_ENABLED: false
+        TESTING_SUPPORT_ENABLED: true
+    global:
+      environment: aat
+      subscriptionId: "1c4f0704-a29e-403d-b719-b90c34ef14c9"
+      tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
+      enableKeyVaults: true
```
Comment (adusumillipraveen) [inline k8s/namespaces/family-public-law/fpl-case-service/aat.yaml:20]: These are not needed, they come from platform defaults when using kustomize.

```suggestion
```

File: k8s/namespaces/family-public-law/fpl-case-service/demo.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      ingressHost: fpl-case-service-demo.service.core-compute-demo.internal
+      environment:
+        GATEWAY_URL: https://gateway-ccd.demo.platform.hmcts.net
+        AUTH_IDAM_CLIENT_BASEURL: https://idam-api.demo.platform.hmcts.net
+        TESTING_SUPPORT_ENABLED: true
+      aadIdentityName: family-public-law
+    global:
+      environment: demo
+      tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
+      enableKeyVaults: true
```
Comment (adusumillipraveen) [inline k8s/namespaces/family-public-law/fpl-case-service/demo.yaml:18]: Same here:
```suggestion

```

File: k8s/namespaces/family-public-law/fpl-case-service/prod.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        GATEWAY_URL: https://gateway.ccd.platform.hmcts.net
+        AUTH_IDAM_CLIENT_BASEURL: https://idam-api.platform.hmcts.net
+        SURVEY_URL_CASESUBMISSION: https://www.smartsurvey.co.uk/s/FamilyPublicLaw?pageurl=application_submission
+        FEES_REGISTER_C2_WITHOUT_NOTICE_KEYWORD: GeneralAppWithoutNotice
+      aadIdentityName: family-public-law
+    global:
+      environment: prod
+      subscriptionId: "8999dec3-0104-4a27-94ee-6588559729d1"
+      tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
+      enableKeyVaults: true
```
Comment (adusumillipraveen) [inline k8s/namespaces/family-public-law/fpl-case-service/prod.yaml:24]: Same here:

```suggestion
   
```

**Review by adusumillipraveen — COMMENTED** [review]
U don't need those defaults

File: k8s/namespaces/family-public-law/fpl-case-service/demo.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+spec:
+  values:
+    java:
+      image: hmctspublic.azurecr.io/fpl/case-service:prod-5a4b25f5
+      ingressHost: fpl-case-service-demo.service.core-compute-demo.internal
+      environment:
+        GATEWAY_URL: https://gateway-ccd.demo.platform.hmcts.net
+        AUTH_IDAM_CLIENT_BASEURL: https://idam-api.demo.platform.hmcts.net
+        TESTING_SUPPORT_ENABLED: true
+      aadIdentityName: family-public-law
```
Comment (adusumillipraveen) [inline k8s/namespaces/family-public-law/fpl-case-service/demo.yaml:14]: You don't need to override this as its defaulted in https://github.com/hmcts/fpl-ccd-configuration/blob/master/charts/fpl-case-service/values.yaml#L11 

```suggestion
 
```

File: k8s/namespaces/family-public-law/fpl-case-service/prod.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+      environment:
+        RELEASE: NOW
+        IDAM_API_URL: https://idam-api.platform.hmcts.net
+        MANAGE_CASE_UI_BASE_URL: https://manage-case.platform.hmcts.net
+        DOCMOSIS_TORNADO_URL: https://docmosis.platform.hmcts.net
+        GATEWAY_URL: https://gateway.ccd.platform.hmcts.net
+        AUTH_IDAM_CLIENT_BASEURL: https://idam-api.platform.hmcts.net
+        SURVEY_URL_CASESUBMISSION: https://www.smartsurvey.co.uk/s/FamilyPublicLaw?pageurl=application_submission
+        FEES_REGISTER_C2_WITHOUT_NOTICE_KEYWORD: GeneralAppWithoutNotice
+      aadIdentityName: family-public-law
```
Comment (adusumillipraveen) [inline k8s/namespaces/family-public-law/fpl-case-service/prod.yaml:19]: Same here:
```suggestion
```

File: k8s/namespaces/family-public-law/fpl-case-service/aat.yaml (hunk 1 above)
Comment (nkoranne) [reply k8s/namespaces/family-public-law/fpl-case-service/aat.yaml:20]: sorted ... added the reference in environment specific customisation.yaml
patchesStrategicMerge:
- ../../../namespaces/family-public-law/fpl-case-service/aat.yaml

File: k8s/namespaces/family-public-law/fpl-case-service/demo.yaml (hunk 4 above)
Comment (nkoranne) [reply k8s/namespaces/family-public-law/fpl-case-service/demo.yaml:14]: removed

File: k8s/namespaces/family-public-law/fpl-case-service/demo.yaml (hunk 2 above)
Comment (nkoranne) [reply k8s/namespaces/family-public-law/fpl-case-service/demo.yaml]: corrected

File: k8s/namespaces/family-public-law/fpl-case-service/prod.yaml (hunk 3 above)
Comment (nkoranne) [reply k8s/namespaces/family-public-law/fpl-case-service/prod.yaml]: corrected

File: k8s/namespaces/family-public-law/fpl-case-service/prod.yaml (hunk 5 above)
Comment (nkoranne) [reply k8s/namespaces/family-public-law/fpl-case-service/prod.yaml]: corrected

**Pull request closed without merging**